server/src/main/resources/db/changelog/20170301083925-add-cert-serial-revoked-column.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="20170301083925-1" author="mstead">
+        <comment>
+            Add cp_cert_serial.revoked column to support removing
+            the hibernate formula off of the CertificateSerial entity.
+        </comment>
+        <addColumn tableName="cp_cert_serial">
+            <column name="revoked" type="BOOLEAN" defaultValue="false">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="20170301083925-2" dbms="postgresql,oracle" author="mstead">
```

Comment:
This should use a single changeset and changeset ID for both mysql & others but should use a property like 20170206114359-convert-quanity-consumed-exported-to-columns.xml. That will ensure that no problem occurs if someone is migrating from mariadb to postgres due to differences in changelog history.